std_msgs/Float32 value
---
std_msgs/Bool success
